.github/workflows/ci-workflow.yml

```diff
@@ -15,3 +18,16 @@ jobs:
       - name: "Run all tests"
         run: |
           make test
+
+  snyk:
+    name: "Run Snyk Workflow"
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Checkout code"
+      uses: actions/checkout@v2
+    - name: "Run Snyk"
+      uses: snyk/actions/golang@master
+      continue-on-error: false
+      with:
+        command: test
+        args: --severity-threshold=high
```

Comment:
Do we need to give it any other arguments like we did for DPC [here](https://github.com/CMSgov/dpc-app/blob/master/.github/workflows/ci-workflow.yml#L54)?

Reply:
No, we only use `go.mod` and this isn't a multi-service repository.